src/MainForm.js

```diff
@@ -73,18 +73,18 @@ class MainForm extends React.Component {
     super(props);
     this.state = { data: data, cols: columns };
     this.handleClear = this.handleClear.bind(this);
-    // this.handleDynamo = this.handleDynamo.bind(this);
   }
 
   handleClear() {
     this.setState = { data: [], cols: [] };
+    // I don't know how to make this work and also re-render after clearing
```

Comment:
you have to call this.setState({data: [], cols: []})

Reply:
soooooooo I'm doing that on line 79 but I don't know how to expose this to header.js, where it's called